tasmota/xnrg_03_pzem004t.ino

```diff
@@ -227,13 +228,14 @@ void PzemEvery250ms(void)
       Pzem.read_state = 0;  // Set address
     }
 
-    Pzem.send_retry = 5;
+    Pzem.send_retry = PZEM_WATHCDOG;
     PzemSend(pzem_commands[Pzem.read_state]);
   }
   else {
     Pzem.send_retry--;
     if ((Energy.phase_count > 1) && (0 == Pzem.send_retry) && (TasmotaGlobal.uptime < PZEM_STABILIZE)) {
       Energy.phase_count--;  // Decrement phases if no response after retry within 30 seconds after restart
+      hass_init_step += PZEM_WATHCDOG/4 + 1; // Don't send Home Assistant discovery yet, delay by 5*250ms + 1s
```

Comment:
What fun if Hass has not been enabled....

Will fail to compile.

Let's make it a TasmotaGlobal variable as I need it too for TasmotaDiscovery, the new and more generic discovery process.

Reply:
Yes. I will add this PR with my proposed global variable.

Just sit back. I'll fix it and add this too.